buildspec.yml

```diff
@@ -0,0 +1,55 @@
+version: 0.2
+
+#      ____        _       _       ____        _ _     _
+#     | __ )  __ _| |_ ___| |__   | __ ) _   _(_| | __| |___ _ __   ___  ___
+#     |  _ \ / _` | __/ __| '_ \  |  _ \| | | | | |/ _` / __| '_ \ / _ \/ __|
+#     | |_) | (_| | || (__| | | | | |_) | |_| | | | (_| \__ | |_) |  __| (__
+#     |____/ \__,_|\__\___|_| |_| |____/ \__,_|_|_|\__,_|___| .__/ \___|\___|
+#                                                           |_|
+#     This file is the codebuild's main entry, which creates a batch run containing
+#     the following 15 runs defined in their own buildspec files.
+#     Default env:
+#       image: aws/codebuild/amazonlinux2-x86_64-standard:3.0
+#       compute-type: BUILD_GENERAL1_SMALL
+#     For other configuration options, see
+#     https://docs.aws.amazon.com/codebuild/latest/userguide/batch-build-buildspec.html
+
+batch:
```

Comment:
it'll be cool, if this can be generated using jinja template 😋. Not a blocker or even necessary.

Reply:
You mean go through the manifest and generate this buildspec automatically? 🤔 
Yeah it is a good idea, but then we will need another codebuild to generate this buildspec.yml and pass it to the build codebuild project...